### PR TITLE
Improve usability

### LINKED ISSUE #1: Not able to generate .json file
I am using, 
swagger-inline './server.js' --base './openapi/openapiBase.yaml' --format '.json' --out './openapi/openapi.json'
in terminal to create the .json file but i am not able to generate the file as described also i am getting the below message 

1 files matched...
0 swagger definitions found...
swagger.json created!

### LINKED ISSUE #2: python 2.7 assumed but not documented
The `utils\gen_py_client.py` file uses the print statement instead of the print() function which will throw an error when using python 3. The OpenAPI experience content only specifies packages like `urllib3`, `certifi`, and `six` and doesn't specify python version.

### PR DESCRIPTION
**Pull request by hickeyma**
Had good discussion with @timroster about usability. Here are the usability issues being covered: 

- Provide inline swagger comment for post api
- Reference the swagger pet store example for user guidance
- Point the user that the visitor object should be in the base openapi
file
- Python code is now 2.7 & 3 compatible (PR #5 )
- Update the base openapi to be standard YAML
- Remove deprecated verification section from challenge 1
- Add note to clear server cache before testing

Closes #2 
Closes #1

### CODE REVIEW

File: docs/draftOpenAPIChallenge.md
Comment (timroster) [inline docs/draftOpenAPIChallenge.md]: With the addition of the example for the /post - the line numbers increment by 22. New numbers are 60, 88, 104 and 134 for the respective endpoints.

Comment (timroster) [reply docs/draftOpenAPIChallenge.md]: Rephrase last sentence to "...comment has already been added for guidance."

Comment (timroster) [reply docs/draftOpenAPIChallenge.md]: I started to add the updates from this file to Influitive, but until server.js is merged this won't work so holding off on that until the PR is merged.

File: docs/generateClientCodeChallenge.md
Comment (timroster) [inline docs/generateClientCodeChallenge.md]: Updated on Influitive, however, I've moved this up into the other dependency note about urllib3, certifi and six as it's best dealt with before running any python code:

"...may include `urllib3`, `certifi` and `six`. The tools provided in the repository are intended for use with Python 2.7."

**Review by henrynash — APPROVED** [review]
LGTM - some nice improvements here.

File: docs/draftOpenAPIChallenge.md
Comment (hickeyma) [reply docs/draftOpenAPIChallenge.md]: Good comments. I updated:
- The `server.js` line numbers
- Changed 'being' to 'been'

Good idea to hold back on updating the coder program, until the PR gets merged.

File: docs/generateClientCodeChallenge.md
Comment (hickeyma) [reply docs/generateClientCodeChallenge.md]: Good idea. Updated.

**Comment by hickeyma** [comment]
@timroster Thanks for review and comments. Updated and ready for review again.

**Comment by hickeyma** [comment]
Updates after rebase. Ready for review.

**Comment by henrynash** [comment]
Still LGTM

**Comment by timroster** [comment]
LGTM

**Comment by timroster** [comment]
Updated IBM Coder (with corrected links)

**Comment by hickeyma** [comment]
Thanks @henrynash for merge.

**Pull request merged**
